examples/pyplots/fig_axes_customize_simple.py

```diff
@@ -32,3 +33,25 @@
     line.set_markeredgewidth(3)
 
 plt.show()
+
+#############################################################################
+#
+# ------------
+#
+# References
+# """"""""""
+#
+# The use of the following functions, methods, classes and modules is shown
+# in this example:
+
+import matplotlib
```

Comment:
So I guess I'm not sure if we should have *every* fcn used by every example listed like this...  Are you curating this at all?

Reply:
I'm curating this a bit indeed. But I may have been inconsistent in doing so in some places.  
In general I'm aiming at putting those functions/methods classes in the references that are important for the example. E.g. if the example treats some complex histogram plotting, I won't put `ax.set_xlabel` in the references. If the example aim is to to show some basic plotting, I'm putting the basic functions in. So far I've also tried to make sure there is at least one example for each new method I came accross, so the whole process surely depends a bit on the order of processing the examples - which is admitedly arbitrary.
But there might be cases where I deviated from that strategy - it's always a bit handwavy.

In this particular case I guess the logic was that the example is rather basic and those methods had not been covered previously *as far as I remember*.

Reply:
That being said, I guess it's beneficial to be slightly more inclusive with references for now. Later, once all examples have a reference section, the aim would be to inversely go through the docs and see if there are overcrowded gallery sections - in which case one can easily remove some references in those examples, as well as see if there are still methods or classes which do not have an example, but should - in which case one can either add a new example or find an existing one which can be modified. 

At the end one benefit of those references will also be to manually check example coverage of the library.